src/menu_events.cpp

```diff
@@ -454,6 +454,29 @@ void menu_handler::show_enemy_moves(bool ignore_units, int side_num)
 			gui_->highlight_another_reach(path);
 		}
 	}
+	
+	//   Fetch cursor position, in order to later find a possible
+	// unit under it.
+	int x = -1;
+	int y = -1;
+	// auto bitmask =
+			SDL_GetMouseState(&x, &y);
+
+	//   Find possible unit (no matter whether friend or foe) under
+	// the cursor.
+	const mouse_handler& mh = const_cast<mouse_handler&>(
+			pc_.get_mouse_handler_base());
+	const map_location hex_under_mouse = mh.hex_clicked_on(x, y);
+	const auto selected_hex_has_unit = mh.hex_hosts_unit(hex_under_mouse);
+    
+	if (selected_hex_has_unit) {
+		//   At this point, a single pixel move would remove the
```

Comment:
Remove excessive indent in comment line.

Reply:
Will comply request.